src/components/single/CommentSection.jsx

```diff
@@ -0,0 +1,14 @@
+import { CommentItem } from './CommentItem';
+
+export const CommentSection = ({comments}) => {
+    return (
+        <section className="comment-article">
+          <h2>Comments</h2>
```

Comment:
Maybe have a loading tag here, incase there's a lag in getting the comments on first render.